### PR TITLE
Rebase/v1.21.0-rc2

### PR DESCRIPTION
**Pull request by adlrocha**
This PR rebases to lotus v1.21.0-rc2

### CODE REVIEW

**Comment by adlrocha** [comment]
@dnkolegov let's not merge it yet even after the review. The rebase introduced a regression when killing test (we are back to `SIGTERM` not killing the test process, it now closes the nodes but doesn't kill the whole process for some reason). @hmoniz will help me look into it before merging

File: .github/ISSUE_TEMPLATE/config.yml
Comment (dnkolegov) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Why do we need all these issue template files?

**Review by dnkolegov — APPROVED** [review]

**Pull request merged**
